packages/client/src/object/convertWireToOsdkObjects/ObjectHolder.ts

```diff
@@ -34,12 +33,10 @@ export interface ObjectHolderPrototypeOwnProps {
   readonly [ClientRef]: MinimalClient;
   readonly "$as": DollarAsFn;
   readonly "$link": ReturnType<typeof get$link>;
-  readonly "$updateInternalValues": (newValues: Record<string, any>) => void;
```

Comment:
Ok so now that we're just using the raw object, we don't need this

Reply:
Yup, this is why I said we have to give up on this functionality to achieve this.